code/modules/food_and_drinks/food/condiment.dm

```diff
@@ -234,6 +234,28 @@
 	list_reagents = list(/datum/reagent/consumable/flour = 30)
 	possible_states = list()
 
+/obj/item/reagent_containers/food/condiment/flour/on_reagent_change(changetype) //born of intense hatred
+	if(!reagents.has_reagent(/datum/reagent/water, 10))
+		return ..()
+	else
```

Comment:
Not needed, remove the else and tab it all out. Returns dont need else statements.